torch/_functorch/functional_call.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Dict, Union, Any, Tuple
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+
+def functional_call(
+    module: 'torch.nn.Module',
+    parameter_and_buffer_dicts: Union[Dict[str, Tensor], Tuple[Dict[str, Tensor]]],
+    args: Union[Any, Tuple],
+    kwargs: Dict[str, Any] = None,
+):
```

Comment:
- I assume the docs are coming later?
- did you want nn.utils.stateless.functional_call to also have this API, or does nn.utils.stateless.functional_call have a different API (and we deprecate it?)

Reply:
Ahh thanks for the catch I forgot to move the docs over

Re: `nn.utils.stateless.functional_call`, I think we should deprecate it and ask users to use torch.func instead since they are the same API. I'll add a PR to do that. Also since it was technically BC-breaking to change this since previously parameters_and_buffers must have been a single dictionary